server/store/alert.go

```diff
@@ -0,0 +1,98 @@
+package store
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+const (
+	splunkAlertKey = "splunkalert"
+	splunkAlertMap = "splunkalertmap"
+)
+
+// AlertStore API for alert KVStore.
+type AlertStore interface {
+	GetChannelIDForAlert(alert string) (string, error)
+	GetChannelAlertIDs(channelID string) ([]string, error)
+	CreateAlert(channelID string, alertsID string) error
+	DeleteChannelAlert(channelID string, alertsID string) error
+}
+
+func keyWithChannelID(channelID string) string {
+	return fmt.Sprintf("%s_%s", splunkAlertKey, channelID)
+}
+
+func (s *pluginStore) GetChannelIDForAlert(alertID string) (string, error) {
+	var alertsMap map[string]string
+	err := s.alertStore.loadJSON(splunkAlertMap, &alertsMap)
+	if err != nil {
+		return "", errors.Wrap(err, "failed to load splunk alerts from store")
+	}
+	if channelID, ok := alertsMap[alertID]; ok {
+		return channelID, nil
+	}
+	return "", nil
+}
+
+func (s *pluginStore) GetChannelAlertIDs(channelID string) ([]string, error) {
+	var alerts []string
+	err := s.alertStore.loadJSON(keyWithChannelID(channelID), &alerts)
+	return alerts, err
+}
+
+func (s *pluginStore) CreateAlert(channelID string, alertID string) error {
+	channelAlerts, err := s.GetChannelAlertIDs(channelID)
+	if err != nil {
+		return errors.Wrapf(err, "failed to get alerts for channel %s", channelID)
+	}
+	var alertsMap = make(map[string]string)
```

Comment:
newlines in this function as well

```suggestion
	channelAlerts, err := s.GetChannelAlertIDs(channelID)
	if err != nil {
		return errors.Wrapf(err, "failed to get alerts for channel %s", channelID)
	}

	var alertsMap = make(map[string]string)
```